packages/components/src/Content/Content.stories.mdx

```diff
@@ -4,7 +4,7 @@ import { Text } from "@jobber/components/Text";
 import { Heading } from "@jobber/components/Heading";
 import { InputText } from "@jobber/components/InputText";
 import { Button } from "@jobber/components/Button";
-import { Tabs, Tab } from "@jobber/components/Tabs";
+import { Tab, Tabs } from "@jobber/components/Tabs";
```

Comment:
This is autosaving in reverse order after I went to this file and copied this line for Typography